mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java

```diff
@@ -16,7 +16,7 @@
 import android.util.Log;
 
 public class ConnectivityChangeReceiver extends BroadcastReceiver {
-    private static final String TAG = "ConnectivityChangeReceiver";
+    private static final String TAG = ConnectivityChangeReceiver.class.getSimpleName();
```

Comment:
I've pushed an updated commit now.
All classes that uses logging has been updated.